crates/sui-core/src/consensus_adapter.rs

```diff
@@ -126,18 +148,6 @@ impl ConsensusAdapterMetrics {
     }
 }
 
-/// Submit Sui certificates to the consensus.
```

Comment:
In terms of this file organization, can we actually keep the order of things as 

* ConsensusAdapter struct definition
* Other non-test structs definition
* ConsensusAdapter implementation
* Other structs implementation
* Test structs definition
* Test structs implementation

This is roughly how we keep other files in sui - main things on top of the file, additional things down below